crates/xilem_html/src/events.rs

```diff
@@ -340,7 +340,7 @@ event_definitions!(
     (OnFocusIn, "focusin", FocusEvent),
     (OnFocusOut, "focusout", FocusEvent),
     (OnFormData, "formdata", Event),
-    (OnInput, "input", InputEvent),
+    (OnInput, "input", Event),
```

Comment:
Ah HTML/DOM is pulling my hair out...

For some reason `<input type=range>` uses the `Event` interface instead of `InputEvent` when using `on_input`, so that's the reason, why this takes the general `Event` type...